ZuluContent/Items/Weapons/BaseWeapon.cs

```diff
@@ -536,10 +536,9 @@ public virtual TimeSpan GetDelay(Mobile m)
             if (speed <= 0)
                 return TimeSpan.FromSeconds(10);
 
-            const double minDelay = 1.25;
+            const double minDelay = 0.5;
 
-            var stamina = Math.Clamp(m.Stam, 0, m.Dex);
-            var delayInSeconds = 15000.0 / ((stamina + 100) * speed);
+            var delayInSeconds = 15000.0 / ((m.Dex + 100) * speed);
```

Comment:
Is this the one you checked vs POL's C++ implementation?

I remember looking at this code from the RunUO.T2A project, it's basically so that you attack slower as you use up your stamina

Reply:
Yes, here it is on POL https://github.com/polserver/polserver/blob/81ad68df2ee5d2de8a69774e70c05526a22e7786/pol-core/pol/mobile/charactr.cpp#L2846